sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobAsyncClient.java

```diff
@@ -978,14 +974,15 @@ public void getBlobTags(String containerName,
                             String blobName,
                             String snapshot,
                             Integer timeout,
+                            String ifTags,
```

Comment:
Is `ifTags` the same as `tagsConditions`? If so, why the name change?

Reply:
Whoops that is meant to be tagsConditions. Yeah they are one and the same, but ifTags doesn't really give a user a good understanding of what it is (ifTags is a SQL like statement that allows a user to execute a request if the blob has the appropriate tags). We had a discussion with the arch board when this feature was released, and tagsConditions was decided as a better name.

Reply:
As for what tags is, it's kind of like a metadata value you can assign to multiple resources to group them

Reply:
https://docs.microsoft.com/en-us/rest/api/storageservices/specifying-conditional-headers-for-blob-service-operations#tags-conditional-operations